docs/source/community_notebooks.md

```diff
@@ -0,0 +1,16 @@
+# Community Tutorials
+
+Community tutorials are made by active members of the Hugging Face community that want to share their knowledge and expertise with others. They are a great way to learn about the library and its features, and to get started with core classes and modalities.
+
+| Task | Class | Modality | Description | Author | Tutorial |
+|------|--------|----------|-------------|---------|----------|
```

Comment:
Maybe it would be clearer to have subsections for the different modalities? Or maybe split by theme?

Eg (a bit shitty, but you get the point

```
## Finetuning VLMs

...

## Making a chatbot

... 
```

Reply:
Sounds good. I went for 'Language Models' and 'Vision Language Models' because it's the simplest.